client-js/app/elements/labrad-registry.ts

```diff
@@ -206,34 +207,41 @@ export class LabradRegistry extends polymer.Base {
     if (event.ctrlKey || event.button == 2 ) {
```

Comment:
I was reading a bit about the DOM drag-n-drop support and it seems like we should be using `event.dataTransfer.dropEffect` here to decide what to do. This should get assigned values by the browser and so should work better cross-platform. For example, on mac, copy is normally triggered using the Alt key, rather than Ctrl. Have you tried checking `dropEffect` here instead?